integration-tests/actions/ocr2vrf_actions/ocr2vrf_steps.go

```diff
@@ -352,10 +321,9 @@ func SetupOCR2VRFUniverse(
 	require.NoError(t, err, "Error setting Payees in VRFBeacon Contract")
 
 	// fund OCR Nodes (so that they can transmit)
-	err = actions.FundChainlinkNodes(chainlinkNodes, chainClient, ocr2vrf_constants.EthFundingAmount)
+	nodes := contracts.ChainlinkK8sClientToChainlinkNodeWithKeysAndAddress(chainlinkNodes)
+	err = actions_seth.FundChainlinkNodesFromRootAddress(l, chainClient, nodes, ocr2vrf_constants.EthFundingAmount)
 	require.NoError(t, err, "Error funding Nodes")
-	err = chainClient.WaitForEvents()
-	require.NoError(t, err, "Error waiting for TXs to complete")
 
 	bootstrapNode := chainlinkNodes[0]
 	nonBootstrapNodes := chainlinkNodes[1:]
```

Comment:
if you really need to modify the original actions please at least move them to `actions_seth` package?

Reply:
@Tofel I don't have legacy vrf code anymore (212c2a3fa2bd058d5ad6b8150ceef5ddde3cd8cc)